puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/builders/gemset_create.yaml

```diff
@@ -0,0 +1,4 @@
+- builder:
+    name: gemset_create
+    builders:
+      - shell: !include-raw scripts/gemset_create.sh
```

Comment:
Good point and I think that could work, before I try that, what are your thoughts on using https://wiki.jenkins-ci.org/display/JENKINS/RVM+Plugin ? Looks like JJB has a macro for it (http://docs.openstack.org/infra/jenkins-job-builder/wrappers.html?highlight=rvm#wrappers.rvm-env).